lib/src/test_runner.dart

```diff
@@ -96,9 +97,8 @@ class GherkinRunner {
     }
 
     await _reporter.dispose();
-    exitCode = allFeaturesPassed ? 0 : 1;
 
-    if (config.exitAfterTestRun) exit(allFeaturesPassed ? 0 : 1);
+    if (config.exitAfterTestRun && allFeaturesPassed) throw 'Tests failed';
```

Comment:
@marcin-jelenski what was the rationale behind this change?

Reply:
@jonsamwell as mentioned above, `dart:io` import in test_runner breaks compatibility with dartjs and breaks tests in dart web-related apps (not flutter-one). There's no `exitCode` there. We're using gherkin for a variety of `angular_dart` tests and that line was a blocker. If you want, I can revert this and prepare a separate PR for this.